### ISSUE TITLE
Latest version number missing 'v' prefix

### ISSUE THREAD

**Issue by borland502** [buluma/ansible-role-influxdb2#20]
## Describe the bug

The error message pretty much sums up the error, but you can work around the problem by specifying the version manually

```console
buluma.influxdb2 was NOT installed successfully: Unable to compare role versions (v0.1.0, v0.1.1, v0.1.2, v0.1.3, v0.1.4, 0.1.5) to determine the most recent version due to incompatible version formats. 
```

## Playbook

Please paste the playbook you are using. (Consider `requirements.yml` and
optionally the command you've invoked.)

```yaml
---
- name: Influxdb2
  hosts: influxdb
  gather_facts: true
  vars_files:
    - secrets/influxdb2.yaml
  vars:
    apt_autostart_state: enabled
    pip_package: python3-pip
    pip_executable: "{{ 'pip3' if pip_package.startswith('python3') else 'pip' }}"
    pip_install_packages:
      - name: setuptools
      - name: requests

  pre_tasks:
    - name: Add new influxdb key
      ansible.builtin.apt_key:
        url: "https://repos.influxdata.com/influxdata-archive_compat.key"
        keyring: "/etc/apt/trusted.gpg.d/influxdata-archive_compat.gpg"

    - name: Add the new influxdb repo
      ansible.builtin.apt_repository:
        repo: "deb [signed-by=/etc/apt/trusted.gpg.d/influxdata-archive_compat.gpg] https://repos.influxdata.com/debian stable main"
        state: present
        install_python_apt: true
        filename: "/etc/apt/sources.list.d/influxdata.list"
        update_cache: true

  roles:
    - role: buluma.bootstrap
    - role: buluma.pip
    - role: buluma.apt_autostart
    - role: buluma.influxdb2
```

## Output

```console
buluma.influxdb2 was NOT installed successfully: Unable to compare role versions (v0.1.0, v0.1.1, v0.1.2, v0.1.3, v0.1.4, 0.1.5) to determine the most recent version due to incompatible version formats. 
```

## Environment

- Control node OS: Fedora Linux 36 (Container Image)
- Control node Ansible version: 2.14.5
- Managed node OS: Debian 11 (bullseye)

**Comment by buluma** [comment]
Thanks mate. fixed tag with latest: v0.1.6

**Comment by borland502** [comment]
And thank you for fixing it.
